apiv2/urls.py

```diff
@@ -97,7 +99,8 @@
 
     # Oauth2
     url(r'^oauth2/', include('apiv2.oauth2_urls', namespace='oauth2_provider')),
-    url(r'^login/$', LoginView.as_view(template_name='api/minimal_login.html'), name="api-login"),
+    url(r'^login/$', login, {'template_name': 'api/minimal_login.html',
+                             'authentication_form': FsAuthenticationForm}, name="api-login"),
```

Comment:
for my interest - does this mean that logins on the API have always been case-sensitive? What difference does this form have to the default one?